linkerd/app/core/src/serve.rs

```diff
@@ -40,38 +40,37 @@ pub async fn serve<M, S, I, A>(
                     };
 
                     // The local addr should be instrumented from the listener's context.
-                    let span = debug_span!("accept", client.addr = %addrs.param());
+                    debug_span!("accept", client.addr = %addrs.param()).in_scope(|| {
+                        let accept = new_accept.new_service(addrs)
```

Comment:
```suggestion
                        let accept = new_accept.new_service(addrs);
```

Reply:
agh i also fixed this but forgot to push it :/